docs/references/javascript/types/passkey-resource.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: PasskeyResource
+description: An interface that describes a passkey associated with a user response.
+---
+
+# `PasskeyResource`
+
+An interface that describes a passkey associated with a user response.
+
+## Properties
+Here is the markdown table:
+
+| Properties                  | Type   | Description                       |
+|-----------------------------|--------|-----------------------------------|
+| `id`                             | `string` | The passkey's unique ID generated by Clerk.                                   |
+| `verification`                   | [`Verification`](/docs/references/javascript/verification#verification) | Verification details for the passkey.                                   |
+| `name`                           | `string` | The passkey's name.                                   |
+| `createdAt`                      | `string` | The date and time when the passkey was created.                                  |
+| `updatedAt`                      | `string` | The date and time when the passkey was updated.                                   |
+| `lastUsedAt`                     | `string` | The date and time when the passkey was last used.                                   |
+
+## Methods
+
+To update or delete a passkey from your code, you must use the `user.passkey.delete` and `user.passkey.update` methods.
+
+For an example of how to use these methods, see the [Passkeys custom flows documentation](/docs/custom-flows/passkeys).
```

Comment:
I feel like this is redundant, we point them to an example of how to use the methods in their respective sections below this text.
We don't have an introductory paragraph for "## Methods" on any other reference page like this, because let's say there were 10 methods. If following suit of this text, there would be a large, redundant intro paragraph.

I think we can cut it.

```suggestion
```